src/borg/repository.py

```diff
@@ -999,7 +1013,7 @@ def segment_exists(self, segment):
     def segment_size(self, segment):
         return os.path.getsize(self.segment_filename(segment))
 
-    def iter_objects(self, segment, include_data=False, read_data=True):
+    def iter_objects(self, segment, offset=0, include_data=False, read_data=True):
```

Comment:
API change ok; all call sites use keyword arguments for the existing 2nd and 3rd arguments.